app/models/apple/podcast_delivery_file.rb

```diff
@@ -11,6 +11,8 @@ class PodcastDeliveryFile < ActiveRecord::Base
 
     delegate :apple_episode_id, to: :podcast_delivery
 
+    PODCAST_DELIVERY_ID_ATTR = "podcast_delivery_id"
```

Comment:
Off topic: just remembering that #469 brings in the Augury rubocop settings (which i think does single quotes?).  I suppose we may want to lint the whole repo sometime after that hits.  Ah well, a problem for future selves.

Reply:
But hey, constants +1!

Reply:
Yeah, there was a rubocop config in feeder that this is using.  When I merge master (and get the new config in place) I can re-lint these.